src/terra-dev-site/app/utilities.2/Logger.app.mdx

```diff
@@ -0,0 +1,47 @@
+# Logger
+
+`terra-application` provides an override-able logger to handle error, info, and warning messages.
+
+## Usage
+
+The logger works like the `console` for the `warn`, `error`, and `info` methods. By default the logger will send the messages to the corresponding console methods and no-op in production for the info and warning messages.
+
+```js
+import logger from 'terra-application/logger';
```

Comment:
```suggestion
import logger from 'terra-application/lib/utils/logger';
```

Reply:
Do we want to make the usage here show a capital-L Logger? Does anyone care?

Reply:
lol, i actually copied what you did in orion-application... it's not really class? so maybe lowercase is fine? 🤷‍♂️

Reply:
I've started picking up the habit/pattern of leaving non-react component directories and filenames lowercase. I vote to leave it lowercase.